src/main/kotlin/com/cognifide/gradle/aem/instance/satisfy/InstanceSatisfy.kt

```diff
@@ -212,9 +201,25 @@ open class InstanceSatisfy : PackageDeploy() {
         return instances.filter { Patterns.wildcard(it.name, group.instanceName) }
     }
 
+    private fun InstanceSync.satisfyPackage(group: PackageGroup, state: PackageState) {
```

Comment:
this allows support for toggling workflows for concrete satisfy group / packages only